.github/workflows/ci.yml

```diff
@@ -11,20 +11,20 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-latest]
+        os: [macOS-latest, ubuntu-latest, windows-latest]
```

Comment:
I just added this since scripted was failing for me locally on mac. I wanted to see if that was the case here as well. However, if you want, I can remove this from the matrix.